### PR TITLE
Add changes to the pipeline to deploy in live cluster

### PR DESCRIPTION
**Pull request by Ramparitala**
Deploy to Staging live and Production live cluster.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -142,23 +145,23 @@ jobs:
...
                     deployment/c100-application-deployment-<< parameters.env_name >> \
                     kubernetes.io/change-cause="${BUILD_DATE} set image ${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1} via CircleCI"
- 
-  deploy_image_to_live:
+
+  deploy_image_to_live_prod:
     <<: *cloud_container
     parameters:
       env_name:
-        type: string
```
Comment (zheileman) [inline .circleci/config.yml:150]: I think is this. Leave it as `string` do not change to `production`.

Comment (Ramparitala) [reply .circleci/config.yml]: Resolved

File: .circleci/config.yml [hunk 2]
```diff
@@ -142,23 +145,23 @@ jobs:
...
       - run:
           name: Authenticate to LIVE cluster
           command: |
-            echo -n ${KUBE_ENV_LIVE_STAGING_CACERT} | base64 -d > ./ca.crt
-            kubectl config set-cluster ${KUBE_ENV_LIVE_STAGING_NAME} --certificate-authority=./ca.crt --server=https://${KUBE_ENV_LIVE_STAGING_NAME}
-            kubectl config set-credentials circleci --token=${KUBE_ENV_LIVE_STAGING_TOKEN}
-            kubectl config set-context ${KUBE_ENV_LIVE_STAGING_NAME} --cluster=${KUBE_ENV_LIVE_STAGING_NAME} --user=circleci --namespace=${KUBE_ENV_LIVE_STAGING_NAMESPACE}
-            kubectl config use-context ${KUBE_ENV_LIVE_STAGING_NAME}
+            echo -n ${KUBE_ENV_LIVE_PRODUCTION_CACERT} | base64 -d > ./ca.crt
+            kubectl config set-cluster ${KUBE_ENV_LIVE_PRODUCTION_NAME} --certificate-authority=./ca.crt --server=https://${KUBE_ENV_LIVE_PRODUCTIONNAME}
```
[the remark this answers is not in the document]
Comment (Ramparitala) [reply .circleci/config.yml]: Resolved

[the remark this answers is not in the document]
Comment (Ramparitala) [reply .circleci/config.yml]: Resolved

**Review by zheileman — APPROVED** [review]
👍

**Pull request merged**